src/flows.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
I like that this is neatly table-driven, but it would really help to have a comment here explaining the data-structure. I had to piece it together bit by bit.

Reply:
I liked the idea of it as a JSON blob because I was pretty much pulling it from a spreadsheet. But ya, comments would be nice, too bad JSON doesn't have those. Would it suffice to outline the structure where we first use it?